.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 docs
 dist
-index.md
+index.md
```

Comment:
In the future please avoid any unnecessary changes, especially in files that are not relevant, as well as needless white space changes.